### PR TITLE
Add Feature Request Issue Template

### PR DESCRIPTION
**Pull request by akestner**
This PR adds a Feature Request GitHub Issue template to the repository. This will make it easier for users to give the project maintainers the information needed to understand and prioritize new features for Karpenter.

### CODE REVIEW

**Review by njtran — COMMENTED** [review]
Great idea. This looks great. I think it's also possible to do this in our repo. Maybe this would be a better place to host this .md?

https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+name: Feature Request
+about: Suggest an idea for the roadmap! The team will review
+title: "[request]: describe request here"
```
Comment (ellistarn) [inline .github/ISSUE_TEMPLATE/feature-request.md:4]: thoughts on "Feature Request" vs "Request"? I'd also make this `Capital Case`

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+labels: Proposed
```
Comment (ellistarn) [inline .github/ISSUE_TEMPLATE/feature-request.md:5]: consider `feature-request`

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
+* If you are interested in working on this issue or have submitted a pull request, please leave a comment
+
+<!-- Thank you for keeping this note for the community -->
+
+<!-- Please keep this note about security disclosures -->
+
+### Security disclosures
+
+If you think you’ve found a potential security issue, please do not post it in the Issues.  Instead, please follow the instructions [here](https://aws.amazon.com/security/vulnerability-reporting/) or [email AWS security directly](mailto:aws-security@amazon.com).
```
Comment (ellistarn) [inline .github/ISSUE_TEMPLATE/feature-request.md:24]: I don't think this is appropriate for a vendor neutral project like Karpenter.

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+about: Suggest an idea for the roadmap! The team will review
+title: "[request]: describe request here"
+labels: Proposed
+assignees: ''
+
+---
+
+<!-- Please keep this note for the community -->
+
+### Community Note
```
Comment (ellistarn) [inline .github/ISSUE_TEMPLATE/feature-request.md:12]: Can we put this at the bottom of the issue template?

File: .github/ISSUE_TEMPLATE/feature-request.md (hunk 1 above)
Comment (akestner) [reply .github/ISSUE_TEMPLATE/feature-request.md:4]: Request is probably better for the near term as I'd prefer to make it easier for more folks to use the template than not.  I'll update

**Comment by ellistarn** [comment]
> Great idea. This looks great. I think it's also possible to do this in our repo. Maybe this would be a better place to host this .md?
> 
> https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository

IMO, Alex is on the money to commit this where he is.

**Comment by akestner** [comment]
@njtran What do you mean "in our repo"? This PR is to add an issue template for the Karpenter repo.  In fact, it was created by following the guide that you linked

**Comment by njtran** [comment]
You're right, that's totally my bad. I thought you were creating this file yourself, rather than following the guide (which states to make a PR like this at the end) 🤦

File: .github/ISSUE_TEMPLATE/feature-request.md (hunk 3 above)
Comment (akestner) [reply .github/ISSUE_TEMPLATE/feature-request.md:24]: Good catch, I'll remove. Might make sense for the AWS Cloud Provider once that gets split out into its own repo

File: .github/ISSUE_TEMPLATE/feature-request.md (hunk 2 above)
Comment (akestner) [reply .github/ISSUE_TEMPLATE/feature-request.md:5]: I think if we broaden the template name to "Request", `feature-request` as an auto-tag doesn't make as much sense. I'll change this to be `Request`

File: .github/ISSUE_TEMPLATE/feature-request.md (hunk 4 above)
Comment (akestner) [reply .github/ISSUE_TEMPLATE/feature-request.md]: Done and done

File: .github/ISSUE_TEMPLATE/feature-request.md (hunk 3 above)
Comment (akestner) [reply .github/ISSUE_TEMPLATE/feature-request.md]: Done

File: .github/ISSUE_TEMPLATE/feature-request.md (hunk 2 above)
Comment (akestner) [reply .github/ISSUE_TEMPLATE/feature-request.md]: Done

**Review by ellistarn — APPROVED** [review]

**Pull request merged**